Stanford/Jumpstart/Install/Content/Importer/ImporterFieldProcessorCustomBody.php

```diff
@@ -20,16 +20,16 @@ class ImporterFieldProcessorCustomBody extends \ImporterFieldProcessor {
   public function process(&$entity, $entity_type, $field_name) {
     if ($entity_type == "node") {
       if (isset($entity->{$field_name}[LANGUAGE_NONE][0]['value'])) {
-        $entity->{$field_name}[LANGUAGE_NONE][0]['value'] = str_replace('/jsa-content', 'https://jsa-content.stanford.edu', $entity->{$field_name}[LANGUAGE_NONE][0]['value']);
+        $entity->{$field_name}[LANGUAGE_NONE][0]['value'] = str_replace('/sites/jsa-content', 'https://jsa-content.stanford.edu/sites/jsa-content/', $entity->{$field_name}[LANGUAGE_NONE][0]['value']);
```

Comment:
There are two possible patterns that we could get from jsa-content

`<img src="/sites/jsa-content/files/styles/medium/picture.jpg" />`
or
`<img src="https://jsa-content.stanford.edu/sites/jsa-content/files/styles/medium/picture.jpg" />`

The str_replace is going to have to check to ensure that the url being replaced isn't absolute. You can do that by matching `="/sites/jsa-content`. Please revise.